ioctl/newcmd/account/account.go

```diff
@@ -160,7 +160,7 @@ func PrivateKeyFromSigner(client ioctl.Client, cmd *cobra.Command, signer, passw
 		if err != nil {
 			return nil, errors.Wrap(err, "invalid HDWallet key format")
 		}
```

Comment:
why is it changed to client?

Reply:
I guess you're talking about `HdwalletMnemonic`.
The main reason is that we don't want to expose `hdwalletConfigFile`, so we add it in `client` struct, which makes all the related function also needs to be inside client interface.